### ISSUE TITLE
Should we add fdb as a separate source type?

### ISSUE THREAD

**Issue by sandorkertesz** [ecmwf/earthkit-data#92]
### Describe the solution you'd like

Currently to read from an fdb stream (with `pyfdb`) we need to do:

```
import pyfdb
import earthkit.data
r = {class': 'od', ...}
r = fdb.retrieve(request)
ds = earthkit.data.from_source("stream", r)
```
It came up during the discussions that we could use "fdb" itself as a source type. So we could write:

```
import earthkit.data
r = {class': 'od', ...}
ds = earthkit.data.from_source("fdb", request)
```

It seems to be a good idea, especially because earthkit-data already has the  "mars" and "cds" source types.

**Comment by tlmquintino** [comment]
yes please. this would be very convinient.
we should have 2:
* fdb
* fdb stream

**Comment by tlmquintino** [comment]
After discussing with @sandorkertesz, we believe its better to create one a **fdb** source and that _stream_ should become a method of the source, to be passed in the kwargs.

**Comment by sandorkertesz** [comment]
Fixed by #93
